tests/test_qb_timeline.py

```diff
@@ -403,6 +407,36 @@ def test_change_before_start_rp(self):
         with pytest.raises(StopIteration):
             next(gen)
 
+    def test_change_before_start_multiple_rps(self):
+        now = datetime.utcnow()
+        back1, nowish = associative_backdate(
+            now=now, backdate=relativedelta(months=1))
+        back7, nowish = associative_backdate(
+            now=now, backdate=relativedelta(months=7))
+        back14, nowish = associative_backdate(
+            now=now, backdate=relativedelta(months=14))
+        org = self.setup_org_qbs(rp_name='v2', retired_as_of=back14)
+        org_id = org.id
+        self.setup_org_qbs(org=org, rp_name='v3', retired_as_of=back7)
+        org = db.session.merge(org)
+        self.setup_org_qbs(org=org, rp_name='v5')
+        self.consent_with_org(org_id=org_id, setdate=back1)
+        user = db.session.merge(self.test_user)
+        gen = ordered_qbs(user)
+
+        # expect everything in v5
+        expect_baseline = next(gen)
+        assert visit_name(expect_baseline) == 'Baseline'
+        assert (
+            expect_baseline.questionnaire_bank.research_protocol.name == 'v5')
+        for n in (3, 6, 9, 15, 18, 21, 30):
```

Comment:
Noting that v5 is every 3 months, and goes out to 60 months, so tests probably should, also

Reply:
well, yes.  but these tests are using bootstrap functions to capture only an isolated detail under test (and often don't match current config - such a change would break and updating that config would be messy).  there are others testing the persistence load and recur functionality, also in isolation.